Detectors/TOF/workflowIO/src/TOFCalibWriterSpec.cxx

```diff
@@ -55,7 +58,7 @@ DataProcessorSpec getTOFCalibWriterSpec(const char* outdef, bool toftpc, bool ad
                                 BranchDefinition<CalibInfosType>{InputSpec{"input", o2::header::gDataOriginTOF, ddCalib, 0},
                                                                  "TOFCalibInfo",
                                                                  "calibinfo-branch-name",
-                                                                 1,
+                                                                 !onlyDia,
```

Comment:
to be sure : do you want to disable this branch for `onlyDia = false`?

Reply:
No, I want to disable it for onlyDia=true. Is it wrong?

Reply:
No, my question was inverted, should be ok.